circuit_knitting/cutting/cutting_evaluation.py

```diff
@@ -64,10 +62,8 @@ def execute_experiments(
         samplers: Sampler(s) on which to run the sub-experiments.
 
     Returns:
-        - A 3D list of length-2 tuples holding the quasi-distributions and QPD bit information
-          for each sub-experiment. The shape of the list is: (``num_unique_samples``, ``num_partitions``, ``num_commuting_observ_groups``)
-        - Coefficients corresponding to each unique subexperiment's
-          sampling frequency
+        - One :class:`~qiskit.primitives.SamplerResult` instance for each partition.
+        - Coefficients corresponding to each unique subexperiment's sampling frequency
```

Comment:
There's something about the word "frequency" that is confusing to me.  Just a suggestion.

```suggestion
        - Coefficients corresponding to each unique subexperiment's contribution to the reconstructed result
```